.gitignore

```diff
@@ -34,3 +34,7 @@ node_modules
 
 # Optional REPL history
 .node_repl_history
+
+# VIM files
```

Comment:
yep issue that before every commit everybody who uses `vim` will have to close all buffers/editor to remove this files from commit. With this files in `.gitignore` it's a bit faster to work